.github/workflows/CI.yml

```
@@ -13,14 +13,9 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - "1.0"
-          - "1.1"
-          - "1.2"
-          - "1.3"
-          - "1.4"
-          - "1.5"
           - "1.6"
-          - "~1.7.0-0"
+          - "1.7"
+          - "~1.8.0-0"
```

Might want to use the latest nightly builds of 1.8 as well?
```suggestion
          - "1.8-nightly"
```

---

AFAIU we would have to change `1.8-nightly` to `1.8` once 1.8 is released while `~1.8.0-0` will keep working?
From https://github.com/julia-actions/setup-julia/blob/b69122e25a41dfa3fc1b74a4e8c21fd842fc57c7/README.md:
- `~1.3.0-0` is a **tilde** version range that includes _all_ pre-releases. It matches all versions `≥ 1.3.0-` and `< 1.4.0`.
- `1.7-nightly` will install the latest nightly build for the upcoming 1.7 release. This version will only be available during certain phases of the Julia release cycle.